rest-api/tools/setup_database.sh

```diff
@@ -50,26 +50,22 @@ fi
 
 source tools/setup_vars.sh
 
+# Get a randomly generated password
+function randpw {
+    new_password=$(< /dev/urandom LC_CTYPE=C tr -dc _A-Z-a-z-0-9_\!\@\#\$\%\^\&\*\(\)-+= | head -c${1:-16};echo;)
```

Comment:
We need to make sure all these characters are OK when used both in SQL and in our DB config. I'd be worried in particular about the slashes.

I don't think we necessarily need all the punctuation symbols in here. A long password of letters and numbers ought to be OK?